### PR TITLE
Option to configure rekor server.

### PR DESCRIPTION
**Pull request by naveensrinivasan**
- Provided an option to configure the rekor server.
- Fixes https://github.com/slsa-framework/slsa-github-generator/issues/372

Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Review by ianlewis — APPROVED** [review]
Thanks!

**Comment by ianlewis** [comment]
I think we might want to update the generic generator as well before closing #372 though.

File: internal/builders/go/main.go [hunk 1]
```diff
@@ -83,7 +88,7 @@ func runProvenanceGeneration(subject, digest, commands, envs, workingDir string)
 	}
 
 	filename := fmt.Sprintf("%s.intoto.jsonl", subject)
-	err = ioutil.WriteFile(filename, attBytes, 0600)
+	err = ioutil.WriteFile(filename, attBytes, 0o600)
```
[the remark this answers is not in the document]
Comment (naveensrinivasan) [reply internal/builders/go/main.go]: took care of it. Thanks

File: internal/builders/go/main.go [hunk 2]
```diff
@@ -101,6 +101,7 @@ func runProvenanceGeneration(subject, digest, commands, envs, workingDir string)
 }
 
 func main() {
+	const defaultRekorAddr = "https://rekor.sigstore.dev"
```
Comment (ianlewis) [inline internal/builders/go/main.go:104]: I was thinking to export this from the signing/sigstore package. something like `sigstore.DefaultRekorAddr`

**Review by laurentsimon — APPROVED** [review]
Thanks

**Pull request merged**